Modules/_threadmodule.c

```diff
@@ -1595,41 +1595,47 @@ PyInit__thread(void)
     timeout_max = floor(timeout_max);
 
     v = PyFloat_FromDouble(timeout_max);
-    if (!v)
-        return NULL;
-    if (PyModule_AddObject(m, "TIMEOUT_MAX", v) < 0)
-        return NULL;
+    if (PyModule_Add(m, "TIMEOUT_MAX", v) < 0)
+        goto error;
 
     /* Add a symbolic constant */
-    d = PyModule_GetDict(m);
     ThreadError = PyExc_RuntimeError;
     Py_INCREF(ThreadError);
```

Comment:
You might use Py_NewRef(). IMO this syntax helps to understand that ThreadError holds a strong reference.

```
    ThreadError = Py_NewRef(PyExc_RuntimeError);
```